pyhoma/models.py

```diff
@@ -134,11 +134,17 @@ def __len__(self) -> int:
 class State:
     __slots__ = "name", "value", "type"
 
-    def __init__(self, name: str, value: str, type: int, **_: Any):
+    def __init__(self, name: str, type: int, value: str = None, **_: Any):
```

Comment:
Value is an optional string.

Reply:
with 
```py
class State:
    __slots__ = "name", "value", "type"

    def __init__(self, name: str, value: Optional[str], type: int, **_: Any):
        self.name = name
        self.value = value
        self.type = DataType(type)
```
I still get the error: `__init__() missing 1 required positional argument: 'value'`
Defaulting value to None did solve that problem